src/main/java/io/orangebeard/client/OrangebeardClient.java

```diff
@@ -94,10 +101,37 @@ public void log(Log log) {
         }
     }
 
+    public void attachment(Attachment attachment) {
+        if (connectionWithOrangebeardIsValid) {
+            HttpEntity<LinkedMultiValueMap<String, Object>> request = multipartLogRequest(attachment);
+            restTemplate.exchange(format("%s/api/v1/%s/log", endpoint, projectName), POST, request, Response.class);
+        } else {
+            LOGGER.warn("The connection with Orangebeard could not be established!");
+        }
+    }
+
     private HttpHeaders getAuthorizationHeaders(String accessToken) {
         HttpHeaders headers = new HttpHeaders();
         headers.setBearerAuth((accessToken));
         headers.setContentType(MediaType.APPLICATION_JSON);
         return headers;
     }
+
+    private HttpEntity<LinkedMultiValueMap<String, Object>> multipartLogRequest(Attachment attachmentLogItem) {
+        LinkedMultiValueMap<String, Object> parts = new LinkedMultiValueMap<>();
+
+        LinkedMultiValueMap<String, String> filePartHeaders = new LinkedMultiValueMap<>();
```

Comment:
Lang leve java 9+! Ik zou de maps zo maken. Verder hier ff wat static imports doen van die headers en gewoon het maptype gebruiken als type, dan is het allemaal een stuk minder verbose :-). 

Map<String, String> filePartHeaders = Map.ofEntries(entry(key, value), entry(key, value);

Reply:
Ja dat probeerde ik, maar die MultiValueMap is een spring-ding en die wil mijn JDK niet vullen met Map.ofEntries()? En als ik er een gewone map van makk lust HttpEntity 'm niet meer

Reply:
oh ja wacht, het is een multivaluemap. Nvm dan.